Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -121,11 +129,40 @@ private void HandleGibTrigger(EntityUid uid, GibOnTriggerComponent component, Tr
             args.Handled = true;
         }
 
+        private void HandleRattleTrigger(EntityUid uid, RattleComponent component, TriggerEvent args)
+        {
+            if (!TryComp<SubdermalImplantComponent?>(uid, out var implanted))
+                return;
+
+            if (implanted.ImplantedEntity == null)
+                return;
+
+            // Gets location of the implant
+            var ownerXform = Transform(uid);
+            var pos = ownerXform.MapPosition;
+            var x = (int) pos.X;
+            var y = (int) pos.Y;
```

Comment:
Why cast as an int over just using the X and Y directly out of curiosity?

Reply:
pos.X and pos.Y are floats so if you try to use them in chat the coordinates will have a lot of unnecessary numbers behind the dot.
Example:
![image](https://github.com/space-wizards/space-station-14/assets/137322659/70c3d075-e1ce-4c6e-b31c-5510dc163580)

Reply:
Fair enough.